Makefile

```diff
@@ -107,11 +107,11 @@ task:
 
 .PHONY: lint
 lint: golangci-lint ## Run golangci-lint linter & yamllint
-	$(GOLANGCI_LINT) run --enable gochecknoglobals
+	$(GOLANGCI_LINT) run
```

Comment:
`make lint` for me fails with:
```
pkg/workers/document/markdown/link_modifier.go:96:1: cyclomatic complexity 26 of func `(*linkModifierRenderer).Render` is high (> 15) (gocyclo)
func (l *linkModifierRenderer) Render(w io.Writer, source []byte, node ast.Node) error {
^
pkg/workers/linkvalidator/validator.go:48:1: cyclomatic complexity 18 of func `(*ValidatorWorker).Validate` is high (> 15) (gocyclo)
func (v *ValidatorWorker) Validate(ctx context.Context, LinkDestination string, ContentSourcePath string) error {
^
cmd/version/version.go:29:5: Version is a global variable (gochecknoglobals)
var Version = "binary was not built properly"
    ^
pkg/workers/document/markdown/link_modifier.go:27:2: bufPool is a global variable (gochecknoglobals)
	bufPool = sync.Pool{
	^
pkg/workers/document/markdown/parser.go:22:2: extensions is a global variable (gochecknoglobals)
	extensions = []goldmark.Extender{
	^
pkg/workers/document/markdown/parser.go:26:2: gmParser is a global variable (gochecknoglobals)
	gmParser = goldmark.New(goldmark.WithExtensions(extensions...), goldmark.WithParserOptions(extension.WithLinkifyURLRegexp(urlRgx)))
	^
pkg/workers/document/document_worker.go:70:2: bufPool is a global variable (gochecknoglobals)
	bufPool = sync.Pool{
	^
cmd/app/cmd.go:36:5: vip is a global variable (gochecknoglobals)
var vip *viper.Viper
    ^
pkg/manifest/manifest.go:322:5: dirToPersona is a global variable (gochecknoglobals)
var dirToPersona = map[string]string{"usage": "Users", "operations": "Operators", "development": "Developers"}
    ^
pkg/manifest/manifest.go:323:5: personaToDir is a global variable (gochecknoglobals)
var personaToDir = map[string]string{"Users": "usage", "Operators": "operations", "Developers": "development"}
    ^
pkg/registry/repositoryhost/export_test.go:3:5: NewResourceURL is a global variable (gochecknoglobals)
var NewResourceURL = new
    ^
pkg/workers/document/markdown/specs_test.go:24:5: gm is a global variable (gochecknoglobals)
var gm = goldmark.New(goldmark.WithRendererOptions(gmhtml.WithUnsafe()), goldmark.WithExtensions(extension.GFM))
    ^
pkg/workers/document/markdown/specs_test.go:27:5: lmr is a global variable (gochecknoglobals)
var lmr = markdown.NewLinkModifierRenderer()
    ^
make: *** [lint] Error 1
```

Do we leave fixing those to another PR?

This also means that the linter is not executed as part of PR which makes it not that useful to me. Because I don't see any failing check on this PR. What is the benefit of the linter if a CI check does not fail on PR-level?